src/api.test.js

```diff
@@ -82,17 +82,15 @@ describe('api', function() {
       dispatchKeyboardEvent('keydown', 'C');
       assert.strictEqual(this.fTwelve.getKeyDownStack(), 'ABC');
     });
-    it('should recognize F12 and attach', function() {
-      dispatchKeyboardEvent('keydown', 'F');
-      dispatchKeyboardEvent('keydown', '1');
-      dispatchKeyboardEvent('keydown', '2');
+    it('should recognize CONTROLF12 and attach', function() {
+      dispatchKeyboardEvent('keydown', 'CONTROL');
+      dispatchKeyboardEvent('keydown', 'F12');
       assert(this.fTwelveAttached());
     });
-    it('should recognize F12 and detach', function() {
+    it('should recognize CONTROLF12 and detach', function() {
       this.fTwelve.attach();
-      dispatchKeyboardEvent('keydown', 'F');
-      dispatchKeyboardEvent('keydown', '1');
-      dispatchKeyboardEvent('keydown', '2');
+      dispatchKeyboardEvent('keydown', 'CONTROL');
+      dispatchKeyboardEvent('keydown', 'F12');
       assert(!this.fTwelveAttached());
     });
```

Comment:
We're not removing the old trigger so we should leave these tests untouched to verify the old hotkey isn't broken. We want new tests for the new functionality. 

For the ctrlKey modifier you won't be able to use the dispatchKeyboardEvent helper. I think it will be like this: 
```js
window.document.dispatchEvent(new KeyboardEvent(keydown, { key: 'F12', ctrlKey: true }))